server/src/routes/api/v1/parkReviewsRouter.js

```diff
@@ -34,4 +34,31 @@ parkReviewsRouter.post("/", async (req, res) => {
   }
 });
 
+parkReviewsRouter.delete("/:reviewId", async (req, res) => {
+  try {
+    const reviewId = req.params.reviewId;
+    await Review.query().deleteById(reviewId);
+    return res.status(204).json({});
+  } catch (error) {
+    return res.status(500).json({ errors: error });
+  }
+});
+
+parkReviewsRouter.patch("/:reviewId", async (req, res) => {
+  try {
+    const parkId = req.params.parkId;
+    const reviewId = req.params.reviewId;
+    const comments = req.body.comments;
+    await Review.query().patchAndFetchById(reviewId, { comments: comments });
+    const park = await Park.query().findById(parkId);
```

Comment:
Line 52 for `patchAndFetchById` is going to patch the record _and_ return it (because of the `fetch`) - so if we're using this query we should save the output to a variable, or just `patch` with `findById` chained on
https://vincit.github.io/objection.js/api/query-builder/mutate-methods.html#patch